src/maps/language.tsx

```diff
@@ -9,51 +9,51 @@ type LanguageSyntaxHighlighterNames = {
 };
 
 const languageLabels: LanguageLabels = {
-  javascript: 'JavaScript',
-  realtime_javascript: 'JavaScript',
-  rest_javascript: 'JavaScript',
-  java: 'Java',
-  realtime_java: 'Java',
-  rest_java: 'Java',
-  kotlin: 'Kotlin',
-  realtime_kotlin: 'Kotlin',
-  rest_kotlin: 'Kotlin',
-  ruby: 'Ruby',
-  realtime_ruby: 'Ruby',
-  rest_ruby: 'Ruby',
-  python: 'Python',
-  realtime_python: 'Python',
-  rest_python: 'Python',
-  php: 'PHP',
-  realtime_php: 'PHP',
-  rest_php: 'PHP',
-  csharp: 'C# .NET',
-  realtime_csharp: 'C# .NET',
-  rest_csharp: 'C# .NET',
-  go: 'Go',
-  realtime_go: 'Go',
-  rest_go: 'Go',
-  cpp: 'C++',
-  realtime_cpp: 'C++',
-  rest_cpp: 'C++',
-  dart: 'Dart',
-  realtime_dart: 'Dart',
-  rest_dart: 'Dart',
-  swift: 'Swift',
-  realtime_swift: 'Swift',
-  rest_swift: 'Swift',
-  objc: 'Objective-C',
-  realtime_objc: 'Objective-C',
-  rest_objc: 'Objective-C',
-  nodejs: 'Node.js',
-  realtime_nodejs: 'Node.js',
-  rest_nodejs: 'Node.js',
-  flutter: 'Flutter',
-  realtime_flutter: 'Flutter',
-  rest_flutter: 'Flutter',
-  android: 'Android',
-  realtime_android: 'Android',
-  rest_android: 'Android',
+  javascript: 'JavaScript v1.2',
```

Comment:
so this is means when there is a new version of the code from the texttile they need to update here as well?

Reply:
Yeah. The ticket said they were not fussed where the labels were, so I added it to the most convenient place I could think of.